openpype/hosts/traypublisher/plugins/create/create_editorial.py

```diff
@@ -216,10 +225,8 @@ def create(self, subset_name, instance_data, pre_create_data):
             ]
         }
         # Create otio editorial instance
-        if AYON_SERVER_ENABLED:
-            asset_name = instance_data["folderPath"]
-        else:
-            asset_name = instance_data["asset"]
+        asset_name = instance_data["asset"]
```

Comment:
Instance data must have `folderPath` in AYON mode.

Reply:
addressed in c326a8bfd4e9032162bde742d13da628a6f70d1c